products/openbsd.md

```diff
@@ -1,237 +1,247 @@
 ---
+# Product level information
+
 title: OpenBSD
 category: os
 tags: bsd-distribution
 iconSlug: openbsd
 permalink: /openbsd
 versionCommand: uname -r
 changelogTemplate: "https://www.openbsd.org/{{'__RELEASE_CYCLE__'|replace:'.',''}}.html"
-activeSupportColumn: false
+
+# Formatting
+
 releaseDateColumn: true
+eolColumn: true
+activeSupportColumn: false
 releaseColumn: false
 
+# eol(x) = releaseDate(x+2), Estimation releaseDate(x) + 1 year -> round to first of next month
+
+# Release Cycles
+
 releases:
--   eol: 2024-11-01
+-   eol: 2024-11-01  # estimated from releaseDate: 2023-10-16
     releaseCycle: "7.4"
     releaseDate: 2023-10-16
 
--   eol: 2024-05-01
+-   eol: 2024-05-01  # estimated from releaseDate: 2023-04-10
     releaseCycle: "7.3"
     releaseDate: 2023-04-10
 
--   eol: 2023-10-16
+-   eol: 2023-10-16  # estimated from releaseDate: 2022-10-20
     releaseCycle: "7.2"
     releaseDate: 2022-10-20
 
--   eol: 2023-04-10
+-   eol: 2023-04-10  # estimated from releaseDate: 2022-04-21
     releaseCycle: "7.1"
     releaseDate: 2022-04-21
 
--   eol: 2022-10-20
+-   eol: 2022-10-20  # estimated from releaseDate: 2021-10-14
     releaseCycle: "7.0"
     releaseDate: 2021-10-14
 
--   eol: 2022-04-21
+-   eol: 2022-04-21  # estimated from releaseDate: 2021-05-01
     releaseCycle: "6.9"
     releaseDate: 2021-05-01
 
--   eol: 2021-10-14
+-   eol: 2021-10-14  # estimated from releaseDate: 2020-10-18
     releaseCycle: "6.8"
     releaseDate: 2020-10-18
 
--   eol: 2021-05-01
+-   eol: 2021-05-01  # estimated from releaseDate: 2020-05-19
     releaseCycle: "6.7"
     releaseDate: 2020-05-19
 
--   eol: 2020-10-18
+-   eol: 2020-10-18  # estimated from releaseDate: 2019-10-17
     releaseCycle: "6.6"
     releaseDate: 2019-10-17
 
--   eol: 2020-05-19
+-   eol: 2020-05-19  # estimated from releaseDate: 2019-04-24
     releaseCycle: "6.5"
     releaseDate: 2019-04-24
 
--   eol: 2019-10-17
+-   eol: 2019-10-17  # estimated from releaseDate: 2018-10-18
     releaseCycle: "6.4"
     releaseDate: 2018-10-18
 
--   eol: 2019-05-03
+-   eol: 2019-05-03  # estimated from releaseDate: 2018-04-02
     releaseCycle: "6.3"
     releaseDate: 2018-04-02
 
--   eol: 2018-10-18
+-   eol: 2018-10-18  # estimated from releaseDate: 2017-10-09
     releaseCycle: "6.2"
     releaseDate: 2017-10-09
 
--   eol: 2018-04-15
+-   eol: 2018-04-15  # estimated from releaseDate: 2017-04-11
     releaseCycle: "6.1"
     releaseDate: 2017-04-11
 
--   eol: 2017-10-09
+-   eol: 2017-10-09  # estimated from releaseDate: 2016-09-01
     releaseCycle: "6.0"
     releaseDate: 2016-09-01
 
--   releaseDate: 2016-03-29
+-   releaseDate: 2016-03-29  # estimated from releaseDate: 2016-03-29
     eol: 2017-04-11
     releaseCycle: "5.9"
```

Comment:
Starting with this entry you have the estimation after the release date instead of the eol, should be:
```suggestion
-   releaseCycle: "5.9"
    releaseDate: 2016-03-29
    eol: 2017-04-11 # estimated from releaseDate: 2016-03-29
```
Please update all below entries also.